src/services/item/plugins/published/schemas.ts

```diff
@@ -21,6 +21,24 @@ const publishEntry = {
   additionalProperties: false,
 };
 
+const publishEntryWithViews = {
+  type: 'object',
+  properties: {
+    id: { type: 'string' },
+    item: {
+      $ref: 'http://graasp.org/items/#/definitions/item',
+    },
+    creator: {
+      $ref: 'http://graasp.org/members/#/definitions/member',
+    },
+    createdAt: { type: 'string' },
+    itemViews: {
+      type: 'array',
+    },
```

Comment:
Do we need to send all the views (as an array that has potentially hundreds of elements) when requesting the collection info ? 
Are we interested in the total sum, in which case it would make sens to only return the total number of views ?

Reply:
@spaenleh 
What if we need the created date to have a sparkline, Shall we only return the created date, or what do u suggest?
views
```
[
{ createdDate: ''},
{ createdDate: ''},
]
```

Reply:
I think that the spark line should be considered as a separate usecase. My justification is as follows: 
- you might want to query for a particular period of time
- you might to get an aggregated result (binning the results into 1h intervals for example)
- etc

So I think this flexibility would be better exposed on a separate endpoint.
I would like to keep the `/informations` endpoint of a collection more "summarized". 
TLDR: this endpoint should return the total number of views. Create a new endpoint to get the sparkline data that allows to pass in more params.